keyboards/zeal65/readme.md

```diff
@@ -0,0 +1,16 @@
+Zeal65
+====
+
+[Zeal65](https://cdn.shopify.com/s/files/1/0490/7329/products/Zeal65_PCB2.jpg)
+
+This is a 65% PCB with per-key RGB LEDs and supports fixed, 1800-like bottom row and split backspace. It was designed for the Zephyr custom keyboard.
+
+Keyboard Maintainer: [Wilba](http://wilba.tech/) and on [github](https://github.com/Wilba6582)
+Hardware Supported: Zeal65 PCB Rev 1
+Hardware Availability: https://zealpc.net/collections/group-buy-pre-orders/products/zephyr
+
+Make example for this keyboard (after setting up your build environment):
+
+    make zeal65:default
+
+See [build environment setup](https://docs.qmk.fm/build_environment_setup.html) then the [make instructions](https://docs.qmk.fm/make_instructions.html) for more information.
```

Comment:
Please change this to the following:

`See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).`